src/schemathesis/schemas.py

```diff
@@ -398,7 +409,8 @@ def get_content_types(self, endpoint: Endpoint, response: GenericResponse) -> Li
         return self.raw_schema.get("produces", [])
 
 
-class OpenApi30(SwaggerV20):  # pylint: disable=too-many-ancestors
+@attr.s()  # pylint: disable=too-many-ancestors
```

Comment:
I believe these two lines can be reverted since there is no need to have this decorator